content/geometry/object-vehicle/vehicle-structure.adoc

```diff
@@ -1056,17 +1053,45 @@ NOTE: Add a figure. This figure might be applicable to all lights on the rear ri
 |===
 
 
+== Grp_Light_Warning_<warning_idx>
 
+This group contains all parts of the vehicle's warning light.
+Warning lights can include various emergency lights, hazard lights, and more.
 
+`<warning_idx>` denotes the index of warning lights. The index entries
+are sorted from right to left in positive y-direction, and from front to rear, starting with 0.
 
-== Grp_Light_Warning_<part_idx>_<warning_idx>
+NOTE: Add a figure on an exemplary rotating warning light on the roof the vehicle.
 
-This group contains all parts of a warning light of the vehicle (part).
-Warning lights can be all kinds of emergency lights, hazard lights, etc.
+.<Table title>
+[%header, cols="20, 80"]
+|===
 
-The `<warning_idx>` is the index for all warning lights counting from right to left (in positive y direction), and from the front to the rear, starting with 0.
+2+^| <Header>
+
+| *Origin*
+| Center of the light element
+
+| *x-axis*
+| Pointing towards the main light emission, or forwards for rotating lights
+
+| *y-axis*
+| Completes the right-handed coordinate system
 
-NOTE: Add a figure on an exemplary rotating warning light on the roof the vehicle (part).
+| *z-axis*
+| Perpendicular to the x-axis, pointing vertically upwards
+|===
+
+
+== Grp_Mirror_Blindspot_Joint_Left_<blindspot_mirror_joint_idx>
+
+This group contains all parts of the main moving part of a blindspot mirror, which adjusts when moved.
```

Comment:
I am not sure, whether I understand the sentence correctly. Does the "main moving part" refer to the complete component consisting of the plastic/metal structure that holds the mirror as well as the mirror glass itself? Or are we only talking about the joint, which connects the plastic/metal structure with the vehicle?

My proposals

- A: "This group contains all parts of the movable structure that holds the blindspot mirror. The blindspot mirror automatically adjusts when the blindspot joint is moved."
- B. "This group contains all parts that connect the blindspot mirror mounting to the vehicle. The blindspot mirror automatically adjusts when the angle of the blindspot joint changes."

Reply:
Proposal A is correct. It contains all movable parts including the plastic or metal housing and the mirror glass itself (as a child node)

Reply:
Integrated feedback